apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/pathAssertions.kt

```diff
@@ -17,6 +17,7 @@ import java.nio.file.Path
  *
  * @since 0.12.0
  */
+@Deprecated("Use the function from the normal jvm module; the jdk8 extension will be removed with 1.0.0", ReplaceWith("ch.tutteli.atrium.api.infix.en_GB.startsWith"))
```

Comment:
sorry, I did not thought about it before, but we can actually just delete the whole jdk8 extension for the infix API. We would have introduced it with 0.12.0 (next version).